app/mixins/route/account/billing.js

```diff
@@ -20,4 +28,22 @@ export default Mixin.create({
       creditCardInfo,
     });
   },
+
+  setupBillingStepSubscriptions(controllerName) {
+    const controller = this.controllerFor(controllerName);
```

Comment:
You can access current controller instance directly via `this.controller`, without the need to know it's name and resolve it.